packages/clay-panel/src/index.tsx

```diff
@@ -91,7 +93,9 @@ const ClayPanel: React.FunctionComponent<IProps> & {
 		>
 			{!collapsable && (
 				<>
-					{displayTitle && (
```

Comment:
Only thing here is that we also need to check first if `displayTitle` even exists. Right now if it doesn't exist, we will still be rendering this...

```jsx
<ClayPanelHeader>
     <span className="panel-title"></span>
</ClayPanelHeader>
```

Previously if we didn't pass in displayTitle, it wouldn't render anything.

Reply:
@kresimir-coko we can't just add `displayTitle &&` to the existing expression since we are using a ternary operator. We need to wrap the whole expression and only evaluate if `displayTitle` exists. Otherwise, we break the existing API since the existing API without `displayTitle` doesn't render anything for the header. If merge this change, those components would now render an empty element

```jsx
<ClayPanelHeader>
	<span className="panel-title"></span>
</ClayPanelHeader>
```